### PR TITLE
Adding slearn

### PR DESCRIPTION
**Pull request by chenxinye**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by chenxinye** [comment]
@conda-forge/help-python This is ready for review, thanks!

**Comment by chenxinye** [comment]
@conda-forge/help-python

**Comment by chenxinye** [comment]
@conda-forge/help-python This is ready for review, thanks!

**Comment by chenxinye** [comment]
@conda-forge-admin, please ping conda-forge/python

File: recipes/slearn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
+  url: https://files.pythonhosted.org/packages/f8/dd/3bd36708e78802bcab553e7b4bfc15a4a433e26fbe5f5911b1f2533fe8cd/slearn-0.2.4.tar.gz
```
Comment (synapticarbors) [inline recipes/slearn/meta.yaml:11]: Can you use the canonical url shown in the example:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

Additionally the version specified above appears not to match the one you have in the url.

File: recipes/slearn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{% set name = "slearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
```
Comment (synapticarbors) [inline recipes/slearn/meta.yaml:10]: You should be able to drop the `fn` line.

File: recipes/slearn/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  sha256: e58e210667b3f96d6ec494778146708daaf3d4cdd318a320694c93db30b4743d
+
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/slearn/meta.yaml:21]: Under `build` it looks like you only need `python >=3.6`, `pip` and `numpy`. The rest are runtime requirements not needed for the building of the package.

File: recipes/slearn/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  home: https://github.com/nla-group/slearn
+  summary: 'A package linking symbolic representation with scikit-learn machine learning for time series prediction.'
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/nla-group/slearn
+
+extra:
+  recipe-maintainers:
+    - nla-group
```
Comment (synapticarbors) [inline recipes/slearn/meta.yaml:56]: Please specify a github user and not an organization for the maintainers list.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/slearn/meta.yaml (hunk 1 above)
Comment (chenxinye) [reply recipes/slearn/meta.yaml]: Thanks Joshua, just replaced it.

File: recipes/slearn/meta.yaml (hunk 2 above)
Comment (chenxinye) [reply recipes/slearn/meta.yaml]: Now it is removed.

File: recipes/slearn/meta.yaml (hunk 3 above)
Comment (chenxinye) [reply recipes/slearn/meta.yaml:20]: Thanks! Modified.

File: recipes/slearn/meta.yaml (hunk 4 above)
Comment (chenxinye) [reply recipes/slearn/meta.yaml]: Thanks, done!

**Comment by chenxinye** [comment]
Hi. @synapticarbors , finish the changes! thanks, this is ready for review again.

**Review by synapticarbors — APPROVED** [review]

**Pull request merged**